src/DevBetterWeb.Core/Entities/Invitation.cs

```diff
@@ -28,5 +44,14 @@ public void Deactivate()
     {
       Active = false;
     }
+
+    public void UpdateUserPingDate()
```

Comment:
If you needed it, you could pass in the datetime variable but have it default to DateTime.Today:

`public void UpdateUserPingDate(DateTime theDate = DateTime.Today)`

Something close to that should work.

Reply:
ah that's smart

Reply:
though I'm not sure I want people to be able to mess with that

Reply:
at least when I'm using the constructor no one can change the date after the invitation is created. Which is probably how it should be

Reply:
That's only reasonable for the CreationDate; the other dates will change.